core/src/main/scala/kafka/security/auth/SimpleAclAuthorizer.scala

```diff
@@ -71,7 +70,8 @@ class SimpleAclAuthorizer extends Authorizer with Logging {
   private var zkUtils: ZkUtils = null
   private var aclChangeListener: ZkNodeChangeNotificationListener = null
 
-  private val aclCache = new scala.collection.mutable.HashMap[Resource, Set[Acl]]
+  private case class VersionedAcls(acls: Set[Acl], zkVersion: Int)
```

Comment:
This should probably be in the `SimpleAclAuthorizer` object since it does not need access to the instance.